### PR TITLE
Store a breakdown of points along with the total for 3rd line support

### PR DESCRIPTION
**Pull request by nickmcmahon01**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/ProtectLevelCalculator.kt [hunk 1]
```diff
@@ -95,6 +98,7 @@ class ProtectLevelCalculator(
       .distinct()
       .filter { it != ComplexityFactor.IOM_NOMINAL }
       .count()
+      .times(2)
```
Comment (markreesmoj) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/ProtectLevelCalculator.kt:101]: Is this a new requirement?

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/ProtectLevelCalculator.kt [hunk 2]
```diff
@@ -104,7 +108,8 @@ class ProtectLevelCalculator(
   ): Int =
     when {
       communityApiClient.getOffender(crn).gender.equals("female", true) -> {
-        getAdditionalFactorsAssessmentComplexityPoints(offenderAssessment) + getBreachRecallComplexityPoints(crn, convictions)
+        (getAdditionalFactorsAssessmentComplexityPoints(offenderAssessment) + getBreachRecallComplexityPoints(crn, convictions))
+          .times(2)
```
Comment (markreesmoj) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/ProtectLevelCalculator.kt:112]: Is this a new requirement?

File: src/main/resources/application.yml [hunk 3]
```diff
@@ -74,3 +74,6 @@ management:
     info:
       cache:
         time-to-live: 2000ms
+
+calculation:
+  version:  1
```
Comment (markreesmoj) [inline src/main/resources/application.yml:79]: We should probably put a note in the README about when to change this, whether we are using semver or whole numbers etc

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/ProtectLevelCalculator.kt (hunk 1 above)
Comment (nickmcmahon01) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/ProtectLevelCalculator.kt:101]: I've moved them from calling method into the callee as they're two separate requirements that we store the results of individually.

Comment (markreesmoj) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/ProtectLevelCalculator.kt:101]: 👍

**Review by markreesmoj — APPROVED** [review]

**Pull request merged**